app_feup/lib/model/entities/Profile.dart

```diff
@@ -1,12 +1,28 @@
 import 'dart:convert';
 
+import 'package:tuple/tuple.dart';
+
+import 'Course.dart';
+
 class Profile {
   String name;
   String email;
   List<Course> courses;
+  String printBalance;
+  String feesBalance;
+  String feesLimit;
 
   Profile({this.name, this.email, this.courses});
 
+  Profile.secConstructor(String name, String email, List<Course> courses, String printBalance, String feesBalance, String feesLimit) {
+    this.name = name;
+    this.email = email;
+    this.courses = courses;
+    this.printBalance = printBalance;
+    this.feesBalance = feesBalance;
+    this.feesLimit = feesLimit;
+  }
+
```

Comment:
The same questions I ask for the class above could be asked for this so the answer to one of them will also solve the other :smiley_cat:

Reply:
:dog: